tests/behat.yml.dist

```diff
@@ -65,7 +65,7 @@ default:
       application:
         base_url: '{{ behat.base_url }}'
         endpoint: '/poetry/notification'
-    Drupal\MinkExtension:
+    Behat\MinkExtension:
```

Comment:
I think this needs to stay `Drupal\MinkExtension` now that we are using the beta channel.

Reply:
YES sorry i was rolling back all the changes to the stable (green) state, it's now fixed!
🎉